### PR TITLE
PP-PT2MEX-Vermont & Mizraim - Vikings all set!

### PR DESCRIPTION
**Pull request by vermontgarcia**
Hey, We had some questions about inheritance so we  reviewed the learning material. I, (Vermont), Still wondering about inheritance and all that stuff. However we could finished with no issues.

### CODE REVIEW

File: starter-code/src/viking.js [hunk 1]
```diff
@@ -1,11 +1,85 @@
 // Soldier
-function Soldier() {}
+function Soldier(health, strength) {
+    this.health = health;
+    this.strength = strength;
+    this.attack = function(){
```
Comment (sbm2391) [inline starter-code/src/viking.js]: use better:
```js
Soldier.prototype.attack = function(damage){
    this.health = this.health - damage;
}
```

File: starter-code/src/viking.js [hunk 2]
```diff
@@ -1,11 +1,85 @@
...
 
 // Viking
-function Viking() {}
+
+Viking.prototype = Object.create(Soldier.prototype);
+
+function Viking(name, health, strength) {
+    this.name = name;
+    Soldier.call(this, health, strength);
+    this.battleCry = function(){
```
Comment (sbm2391) [inline starter-code/src/viking.js]: use better:
```js
Viking.prototype.battleCry = function() {
  // ...
}
```

File: starter-code/src/viking.js [hunk 3]
```diff
@@ -1,11 +1,85 @@
 // Soldier
-function Soldier() {}
+function Soldier(health, strength) {
+    this.health = health;
+    this.strength = strength;
+    this.attack = function(){
+        return this.strength;
+    }
```
Comment (ta-web-mex) [inline starter-code/src/viking.js]: use better:
```js
Soldier.prototype.attack = function() {
  // ...
}
```

File: starter-code/src/viking.js (hunk 2 above)
Comment (ta-web-mex) [inline starter-code/src/viking.js]: same:
```js
Viking.prototype.battleCry = {
  // ...
};
```

**Pull request closed without merging**